pages/Configuring/Window-Rules.md

```diff
@@ -27,65 +28,77 @@ windowrule=move 0 0,title:^(Firefox)(.*)$
+| animation \[style\] (\[opt\]) | forces an animation onto a window, with a selected opt. Opt is optional.                                                                                                                                                                                                                                               |
+| rounding \[x\]                | forces the application to have X pixels of rounding, ignoring the set default (in `decoration:rounding`). Has to be an int.                                                                                                                                                                                            |
+| noblur                        | disables blur for the window                                                                                                                                                                                                                                                                                           |
+| nofocus                       | disables focus to the window                                                                                                                                                                                                                                                                                           |
+| forceinput                    | forces an XWayland window to receive input, even if it requests not to do so. (Might fix issues like e.g. Game Launchers not receiving focus for some reason)                                                                                                                                                          |
+| pin                           | pins the window _note: floating only_                                                                                                                                                                                                                                                                                  |
+| noanim                        | disables the animations for the window                                                                                                                                                                                                                                                                                 |
+
+### Example Rules
+
 ```ini
 windowrule = move 100 100,^(kitty)$
 windowrule = animation popin,^(kitty)$
 windowrule = noblur,^(firefox)$
 ```
 
+## Notes
+
 {{< hint type=tip >}}
 
-Opacity is always a PRODUCT of all opacities. E.g. active_opacity to
+Opacity is _always_ a PRODUCT of all opacities. E.g. active*opacity to
```

Comment:
It's `active_opacity`. Maybe you can add backticks to prevent the editor from confusing it or something.

Reply:
Good idea.